hugo/content/prometheus/_index.md

```diff
@@ -147,14 +151,31 @@ sudo systemctl start alertmanager
 sudo systemctl status alertmanager
 ```
 
-## Note for packaging (RHEL/CentOS 7)
-
-The service override files must be placed in the relevant drop-in folder to override the default service files.
-
-```
-/etc/systemd/system/prometheus.service.d/crunchy-prometheus-service.conf
-/etc/systemd/system/alertmanager.service.d/crunchy-alertmanager-service.conf
-```
-
-After a daemon-reload, systemd should automatically find these files and the crunchy services should work as intended.
-
+### Setup Windows Server 2012R2
+
+Currently the Windows installers assume you are logged in as the local Administrator account, so please ensure to do so before attempting the following.
+
+Install the Prometheus, AlertManager, and Grafana servers by:
+
+1. Find and double-click the `pgMonitor_server_1.0_Crunchy.win.x86_64.exe` file previously obtained from Crunchy Data. It will present you with the following screen. Choose your install path and click 'Install'.
+    ![server_installer_1.png](/images/server_installer_1.png)
+2. Once installation has finished, clicked 'Close':
+    ![server_installer_2.png](/images/server_installer_2.png)
+3. The installer will launch the Windows services that were just installed. Click 'OK' to proceed:
+    ![server_installer_3.png](/images/server_installer_3.png)
+4. You will now be prompted to launch the configuration tool. Select 'Yes' to continue:
+    ![server_installer_4.png](/images/server_installer_4.png)
+5. Select '1' to tell Prometheus about the exporters it should scrape metrics from:
+    ![server_installer_5.png](/images/server_installer_5.png)
+6. Enter the hostname (just the hostname, not the FQDN) of the PostgreSQL server that the exporters are running on. Next, enter the IOP address of the PostgreSQL server, and the WMI port (default is 9182):
+    ![server_installer_6.png](/images/server_installer_6.png)
+7. Enter a cluster name. This should be something simple but meaningful to identify the PostgreSQL cluster in question, e.g. payroll. Then enter the port used for both the cluster/global `postgres_exporter` (9187 by default) and the per-db `postgres_exporter` (9188 in our directions):
+    ![server_installer_7.png](/images/server_installer_7.png)
+8. You can now choose '2' to exit the configuration tool:
+    ![server_installer_8.png](/images/server_installer_8.png)
+9. You can nowk verify that Prometheus is running by loading [http://localhost:9090](http://localhost:9090) in your browser:
+    ![server_installer_9.png](/images/server_installer_9.png)
+10. Finally, verify Prometheus can access the exporters by choosing 'Status' and then 'Targets':
+    ![server_installer_10.png](/images/server_installer_10.png)
+11. You should show three exporters, all with a green 'Up' status:
```

Comment:
three or " configured " ?

Reply:
good point

Reply:
Yeah probably best not to give an explicit number